### ISSUE TITLE
Segfault on start because vlc-qt tries to load QtGui.so.4

### ISSUE THREAD

**Issue by akien-mga** [MBach/Miam-Player#9]
I had already reported this issue previously but in a pull request (#7), thus it did not have much visibility. Creating a dedicated issue for it now, since I've just build Miam-Player 0.7.0 and the issue is the same.

Backtrace:
```
(gdb) run
Starting program: /home/akien/Mageia/Sandbox/miam-player/Miam-Player-build/code/MiamPlayer/MiamPlayer 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".
[New Thread 0x7fffe2cbe700 (LWP 3592)]
[New Thread 0x7fffe0d1c700 (LWP 3593)]

Program received signal SIGSEGV, Segmentation fault.
0x00007fffdb07fa2c in _GLOBAL__sub_I_qprinterinfo.cpp () from /lib64/libQtGui.so.4
```
```
(gdb) bt
#0  0x00007fffdb07fa2c in _GLOBAL__sub_I_qprinterinfo.cpp () at /lib64/libQtGui.so.4
#1  0x00007ffff7dec26a in call_init (l=<optimized out>, argc=argc@entry=1, argv=argv@entry=0x7fffffffd9d8, env=env@entry=0x7fffffffd9e8) at dl-init.c:78
#2  0x00007ffff7dec37b in _dl_init (env=0x7fffffffd9e8, argv=0x7fffffffd9d8, argc=1, l=<optimized out>) at dl-init.c:36
#3  0x00007ffff7dec37b in _dl_init (main_map=main_map@entry=0xb32540, argc=1, argv=0x7fffffffd9d8, env=0x7fffffffd9e8) at dl-init.c:126
#4  0x00007ffff7df03d1 in dl_open_worker (a=a@entry=0x7fffffffce78) at dl-open.c:566
#5  0x00007ffff7dec114 in _dl_catch_error (objname=objname@entry=0x7fffffffce68, errstring=errstring@entry=0x7fffffffce70, mallocedp=mallocedp@entry=0x7fffffffce67, operate=operate@entry=0x7ffff7deff60 <dl_open_worker>, args=args@entry=0x7fffffffce78) at dl-error.c:187
#6  0x00007ffff7defb63 in _dl_open (file=0xb47fd0 "/usr/lib64/vlc/plugins/gui/libqt4_plugin.so", mode=-2147483647, caller_dlopen=0x7fffefbe6b6f <module_Load+31>, nsid=-2, argc=<optimized out>, argv=<optimized out>, env=0x7fffffffd9e8) at dl-open.c:650
#7  0x00007ffff212e039 in dlopen_doit (a=a@entry=0x7fffffffd090) at dlopen.c:66
#8  0x00007ffff7dec114 in _dl_catch_error (objname=0x8c9ac0, errstring=0x8c9ac8, mallocedp=0x8c9ab8, operate=0x7ffff212dfe0 <dlopen_doit>, args=0x7fffffffd090) at dl-error.c:187
#9  0x00007ffff212e69d in _dlerror_run (operate=operate@entry=0x7ffff212dfe0 <dlopen_doit>, args=args@entry=0x7fffffffd090) at dlerror.c:163
#10 0x00007ffff212e0d1 in __dlopen (file=<optimized out>, mode=<optimized out>) at dlopen.c:87
#11 0x00007fffefbe6b6f in module_Load () at /lib64/libvlccore.so.7
#12 0x00007fffefbd08ad in module_InitDynamic () at /lib64/libvlccore.so.7
#13 0x00007fffefbd0bb7 in AllocatePluginDir () at /lib64/libvlccore.so.7
#14 0x00007fffefbd0e2b in AllocatePluginDir () at /lib64/libvlccore.so.7
#15 0x00007fffefbd0f89 in AllocatePluginPath () at /lib64/libvlccore.so.7
#16 0x00007fffefbd1631 in module_LoadPlugins () at /lib64/libvlccore.so.7
#17 0x00007fffefb68f4a in libvlc_InternalInit () at /lib64/libvlccore.so.7
#18 0x00007fffefe4273b in libvlc_new () at /lib64/libvlc.so.5
#19 0x00007ffff434068c in VlcInstance::VlcInstance(QStringList const&, QObject*) () at /lib64/libvlc-qt.so.0.9
#20 0x00007ffff78bb9a1 in MediaPlayer::MediaPlayer(QObject*) (this=0xb03800, parent=<optimized out>) at mediaplayer.cpp:22
#21 0x00000000004c46e7 in MainWindow::MainWindow(QWidget*) (this=0x8ffe40, parent=<optimized out>) at mainwindow.cpp:31
#22 0x00000000004379ea in main(int, char**) (argc=1, argv=0x7fffffffd9d8) at main.cpp:42
```

As pointed out by @MBach, this issue would be linked to https://github.com/ntadej/vlc-qt/issues/14

System: Mageia 5, 64bit, with vlc-qt 0.9.0.

**Comment by akien-mga** [comment]
Closing, this seems indeed to be a packaging issue for which @MBach can't do much.

**Comment by MBach** [comment]
Nice ! I knew it :D

**Comment by akien-mga** [comment]
Just a heads-up, the packaging issue with VLC was fixed in Mageia when we updated to VLC 2.2.0 RC2, so I could finally import Miam-Player :-)
http://mageia.madb.org/package/show/release/cauldron/application/0/name/miam-player
